lib/rubocop/cop/rspec/mixin/css_selector.rb

```diff
@@ -87,21 +57,11 @@ def attribute?(selector)
         def attributes(selector)
           selector.scan(/\[(.*?)\]/).flatten.to_h do |attr|
             key, value = attr.split('=')
+            value << ']' if value&.include?('[')
```

Comment:
This does not look right. Probably a shortcoming of us using regular expressions to parse CSS :-)